### PR TITLE
Add Swift category and my open source project

### PR DESCRIPTION
**Pull request by mugbug**

### CODE REVIEW

**Review by ohager — APPROVED** [review]
Nice one! Thanks for extending the contributions.md - I need to clarify some details on it.

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Dextra is a Software development company which is really proud of the great work
  - [JavaScript](#javascript-)
  - [.NET](#net-)
  - [TypeScript](#typescript-)
-
+ - [Swift](#swift-)
```
Comment (erickzanardo) [inline README.md:32]: Lets keep this in alphabetic order

**Review by erickzanardo — CHANGES_REQUESTED** [review]
Just fix the order on the index please, otherwise, LGTM

File: README.md (hunk 1 above)
Comment (ohager) [reply README.md:32]: Legit!

Comment (mugbug) [reply README.md:30]: Done 👍

**Review by erickzanardo — APPROVED** [review]
LGTM! Thanks for the contribution!

**Pull request merged**